agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/templates/notification/mail/EventMailPlugin.gtmpl

```diff
@@ -106,7 +122,10 @@
                                                         text-align: center;
                                                         vertical-align: middle;"
                                                         href="<%= tentativeResponseURL %>">
-                                                        <%= _ctx.appRes("Notification.agenda.event.tentative") %>
+                                                        <%
+                                                        if (org.exoplatform.agenda.constant.AgendaEventModificationType.ADDED.name().equals(modificationType)) { %>
```

Comment:
Here you need to test on event status only isn't it ? If it's confirmed, then display three buttons, else not.

Reply:
i will check if the modification type != deleted,